about.html

```diff
@@ -0,0 +1,233 @@
+          <a class="campaign" href="#headline">DW Campaign</a>
+        </ul>
+      </nav>
+      <!--EOF Main Navigation NAV-->
+    </div>
+    <!--EOF .contain-nav-->
+  </header>
+
+  <!--MAIN CONTENT-->
+  <main id="headline">
+    <div class="banner">
+      <div class="container">
+        <p class="slogan mobile">
+          < Learn. Connect. Experience />
+        </p>
+        <h1 class="event-title">DEVELOPER WEEK<br>GLOBAL SUMMIT 2022</h1>
+        <hr class="desktop">
+        <p class="event-text mobile">
+          DevWeek puts the spotlight on new technologies. Companies that
+          participated in past DevWeek events include Google,
+          Facebook, Yelp, Rackspace, IBM, Cloudera, Red Hat, Optimizely,
+          SendGrid, Blackberry, Microsoft, Neo Technology, Eventbrite,
+          Klout, Built.io, Ripple, GNIP, Tagged, HackReactor, and 30+
+          more here!
+        </p>
+        <p class="event-text desktop">
+          DevWeek 2022 is the largest developer conference & event series
+          including the DevWeek 2022 Conference & Expo, 1,000+ attendee
+          hackathon, 800+ attendee tech hiring expo, and a series of
+          workshops, technical talks, and keynotes.<br>
+          Because DevWeek covers all new technologies, our conference
+          and workshops invite you to get intro lessons (or advanced tips
+          and tricks) on technologies like VR Dev, Artificial Intelligence
+          Dev, Blockchain Dev, IoT Dev, Serverless technology, Microservices
+          technology, new JavaScript frameworks, and more.
+        </p>
+        <p class="event-place">
+          Please contact us per Email for any further questions
+          about DW Global Summit 2022!
+        </p>
+        <a id="contact" class="email-link" href="mailto:dwsummit2022@devweek.dev">dwsummit2022@devweek.dev</a>
+
+      </div>
+      <!--EOF .container -->
+    </div>
+    <!--EOF .banner -->
+  </main>
+
+  <!--LOGO ABOUT-->
+  <section id="logo-about">
+    <div class="container">
+      <h2 class="section-title">DevWeek Logo <br>How was DW Logo chosen</h2>
+      <hr>
+      <p class="about-text">
+        The logo of DevWeek was decided through the logo competition from 10.Jan to 10.Feb.
+      </p>
+      <p class="about-text desktop">
+        After online public voting and Selection Committee's judging process,
+        the logo created by Justin Leitner won the prize.
+      </p>
+      <p class="about-text desktop">After consultation with the Selection Committee we optimized the
+        prize-winning-logo, so it could be the Oficial DW Logo for the rest
+        of the year or until another public competition is open.
+      </p>
+      <div class="show-logo">
+        <img src="./assets/logos/devweek-logo.png" alt="DevWeek Logo">
+      </div>
+      <p class="about-text">
+        <small>Please do not edit, change, distort, recolor, or reconfigure the DW Logo.</small>
+      </p>
+    </div>
+  </section>
```

Comment:
- [ ] kindly reduce the space between the paragraph lines to match the required design 🙏 

your design : 
![image](https://user-images.githubusercontent.com/89282221/204111543-e9921816-114a-4fcd-a658-4896de7e2b40.png)

required design : 

![image](https://user-images.githubusercontent.com/89282221/204111589-49de9689-132e-4ee1-a7b2-96cba00d73d6.png)

Reply:
Thanks for the feedback, I've fixed this, do you think that would be OK?
![image](https://user-images.githubusercontent.com/67355479/204112054-b5594729-a50b-4753-a634-bb0532c4c22a.png)

Thanks again for your kindly support!
Cheers,
Luis

Reply:
It should be fine , as soon as you finish the required changes please request another review 🙏